networkx/algorithms/connectivity/edge_kcomponents.py

```diff
@@ -239,7 +239,7 @@ def bridge_components(G):
 class EdgeComponentAuxGraph:
     r"""A simple algorithm to find all k-edge-connected components in a graph.
 
-    Constructing the AuxillaryGraph (which may take some time) allows for the
+    Constructing the AuxiliaryGraph (which may take some time) allows for the
```

Comment:
This should also not be a CamelCase word (which indicates a class)...  Could we add a space and go lowercase on AuxiliaryGraph -> auxiliary graph